### PR TITLE
Cleanup download page

### PR DESCRIPTION
**Pull request by Chocobo1**
I also converted download page to Markdown...

@sledgehammer999 
The markdown change can be throw away if you think just HTML is good enough.

### CODE REVIEW

**Comment by sledgehammer999** [comment]
IMO, markdown should be used where we expect to edit the page frequently. (eg news). Other places, should stay HTML. So the download page should stay in HTML.

**Comment by Chocobo1** [comment]
OK, I ditched the .md format.

**Comment by Chocobo1** [comment]
If you have time, can you take a glimpse?

**Comment by sledgehammer999** [comment]
Is it possible to post a screenshot?

**Comment by Chocobo1** [comment]
screenshot

File: download.php [hunk 1]
```diff
@@ -1,164 +1,260 @@
...
+<p>
+  Starting from v3.3.4 all source tarballs and binaries are signed.
+</p>
+<p>
+  The key currently used is: <a href="https://pgp.mit.edu/pks/lookup?op=get&amp;search=0x6E4A2D025B7CC9A2"><code>4096R/5B7CC9A2</code></a><br/>
+  Fingerprint: <code>D8F3DA77AAC6741053599C136E4A2D025B7CC9A2</code><br/>
+  You can also download it from <a href="https://github.com/qbittorrent/qBittorrent/raw/master/5B7CC9A2.asc">here</a>
+</p>
+<p>
+  Previously the following key was used to sign the v3.3.4 source tarballs and v3.3.4 Windows installer <em>only</em>: <a href="https://pgp.mit.edu/pks/lookup?op=get&amp;search=0xA1ACCAE4520EC6F6"><code>4096R/520EC6F6</code></a><br/>
```
Comment (sledgehammer999) [inline download.php]: I want the word `only` to be bold.

**Comment by sledgehammer999** [comment]
The changes look very good in the screenshot. Only one minor thing I want changed.
PS: I didn't look closely to the other-os changes. I trust that you did it correctly.

You are free to merge as soon as you bold the `only` word.

File: download.php (hunk 1 above)
Comment (Chocobo1) [inline download.php]: fixed.

**Comment by Chocobo1** [comment]
Thanks for the feedback, if there are other issues, we can fix it in no time.

**Pull request merged**